<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zjw_brepClass</class>
 <widget class="QMainWindow" name="zjw_brepClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zjw_brep</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="ZjwOpenGL" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>0</y>
      <width>761</width>
      <height>581</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>-1</y>
      <width>131</width>
      <height>491</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Brep">
     <attribute name="title">
      <string>Brep</string>
     </attribute>
     <widget class="QLabel" name="renderModel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>render model</string>
      </property>
     </widget>
     <widget class="QLabel" name="solidModel">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>solid model</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="solid1RadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>solid 1</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="solid2RadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>solid 2</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="solid3RadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>solid 3</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="solid4RadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>solid 4</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="wireFrameRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>121</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>wire-frame model</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="faceRadioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>89</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>face model</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="other">
     <attribute name="title">
      <string>other</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ZjwOpenGL</class>
   <extends>QOpenGLWidget</extends>
   <header>zjw_opengl.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="zjw_brep.qrc"/>
 </resources>
 <connections/>
</ui>
